### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Local IIS Server_Keyboard: Keyboard focus is incorrect, it is going  to more command button(...) instead of Local iss link

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#60]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0), Chrome(68.0.3440.106)

Keyboard.
MAS ID Violated: MAS 2.4.3,MAS 1.3.2

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Local IIs link
4.Check whether the focus order is correct or not

Actual Result:
Keyboard focus is incorrect, it is going  to more command button(...) instead of Local IIS  link

Expected Result:
Key board focus should move first to Local IIS link then it should move to More(...) button

User Impact:
Keyboard only users find difficulty in using the application if  focus order is incorrect 

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

**Comment by Amlan454** [comment]
[Bug_60_Focus order is incorrect.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317181/Bug_60_Focus.order.is.incorrect.pptx)

**Comment by jhkimnew** [comment]
Fixed.

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.4.3; #C+E-H; #.NETWebsites; #Chrome; #Edge; #MAS1.3.2; #ScreenReader; #Narrator; #NVDA

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
